.storybook/babel.plugin.js

```diff
@@ -0,0 +1,32 @@
+// This Babel plugin adds context.parameters.fullSource property to Storybook stories,
+// which contains source of of the file where story is present.
+// It’s utilized by Export to CodeSandbox.
+
+// @ts-ignore
+module.exports = function (babel) {
+  const { types: t } = babel;
+  return {
+    name: 'literal-replacer',
+    visitor: {
+      // @ts-ignore
```

Comment:
do we need a ts-ignore in a js file ?

Reply:
Yeah.. without the ts-ignore I get a typescript error in vs code. I don’t understand why.
`Parameter 'path' implicitly has an 'any' type."`

Reply:
`.storybook/tsconfig.json` has `checkJs: true`. An alternative instead of adding ts-ignores is to either declare the proper types, or add `noImplicitAny: false` to that tsconfig.

I checked out the branch locally and you can get the proper types/inference like this:
```js
/**
 * This Babel plugin adds `context.parameters.fullSource` property to Storybook stories,
 * which contains source of of the file where story is present.
 * It’s utilized by Export to CodeSandbox.
 * @param {import('@babel/core')} babel
 * @returns {import('@babel/core').PluginObj}
 */
module.exports = function (babel) {
  const { types: t } = babel;

  return {
    name: 'literal-replacer',
    visitor: {
      MemberExpression(path) {
        if (
          t.isIdentifier(path.node.property) &&
          t.isIdentifier(path.node.object) &&
          path.node.property.name === 'parameters' &&
          path.parentPath.isAssignmentExpression()
        ) {
          const storyName = path.node.object.name;
          const expression = t.expressionStatement(
            t.assignmentExpression(
              '=',
              t.memberExpression(
                t.memberExpression(t.identifier(storyName), t.identifier('parameters')),
                t.identifier('fullSource'),
              ),
              t.identifier('__STORY__'),
            ),
          );
          const expressionStatement = path.findParent(p => p.isExpressionStatement());
          expressionStatement.insertAfter(expression);
          path.stop();
        }
      },
    },
  };
};
```